api/tests/routes/backoffice_v3/collective_offers_test.py

```diff
@@ -501,3 +505,205 @@ def test_get_batch_collective_offers_reject_form(self, legit_user, authenticated
 
         # then
         assert response.status_code == 200
+
+
+class GetCollectiveOfferDetailTest(GetEndpointHelper):
+    endpoint = "backoffice_v3_web.collective_offer.collective_offer_details"
+    endpoint_kwargs = {"collective_offer_id": 1}
+    needed_permission = perm_models.Permissions.READ_OFFERS
+
+    def test_nominal(self, legit_user, authenticated_client):
+        # when
+        pricing = finance_factories.CollectivePricingFactory()
+        url = url_for(self.endpoint, collective_offer_id=pricing.collectiveBooking.collectiveStock.collectiveOffer.id)
+        response = authenticated_client.get(url)
```

Comment:
```suggestion
        with assert_num_queries(3):
            response = authenticated_client.get(url)
```
ainsi que pour les fonctions suivantes.
Éventuellement 4 lorsque ça appelle la vérification d'"éditabilité".